servo/runner.py

```diff
@@ -284,6 +288,12 @@ async def connect() -> None:
         else:
             self.logger.warning(f"Servo runner initialized with polling disabled -- command loop is not running")
 
+        if not servo.current_servo().config.no_diagnostics:
+            diagnostics_handler = servo.telemetry.DiagnosticsHandler(self.servo)
+            self._diagnostics_loop_task = asyncio.create_task(diagnostics_handler.diagnostics_check(), name=f"diagnostics for servo {self.optimizer.id}")
+        else:
+            self.logger.info(f"Servo runner initialized with diagnostics polling disabled")
+
```

Comment:
Hmm, it was there originally but moved it to the run based on [first review comment](https://github.com/opsani/servox/pull/373#discussion_r766001691)? What do you think is best?

Reply:
You're right, my bad. I think it can be left where it is and instead just remove the task cancellation as part of the main loop startup (or move the cancellation to where the task is being created currently) https://github.com/opsani/servox/blob/ff9ec7b1b305037c3afd984ae124e69b5f6a6ef1/servo/runner.py#L226-L227

Reply:
I do actually prefer it where it was originally so both loops are on the same tier